zproject/settings.py

```diff
@@ -1397,7 +1397,7 @@ def get_dirs(self):
     'feedback@zulip.com',
     'notification-bot@zulip.com',
     'welcome-bot@zulip.com',
-    'new-user-bot@zulip.com',
+    'default-bot@zulip.com',
```

Comment:
This should not be here -- please remove it.

Reply:
Got it, removed.